Documentation/how-to-build-and-run-ilcompiler-in-visual-studio.md

```diff
@@ -34,6 +34,8 @@ _Note: The size of NuGet packages is approximately 2.75 GB, so download might ta
 
   - The repro project has a dummy program that you can modify for ad-hoc testing
 
+  - If you want to write your repro code using IL (instead of C#), you can search/replace `"repro\repro.csproj"` to `"repro\repro.ilproj"` in ILCompiler.sln and use repro.ilproj instead. Everything else below still applies. Please don't check this in.
```

Comment:
I would add this note at the very end of this doc. It is advanced stuff that should not distract folks going through this for the first time. Also, it applied equally for both codegens.